app/lib/pages/memories/sync_page.dart

```diff
@@ -280,26 +280,27 @@ class _SyncPageState extends State<SyncPage> {
     }
 
     return PopScope(
-      canPop: !Provider.of<MemoryProvider>(context, listen: false).isSyncing,
+      canPop: true,
       onPopInvoked: (didPop) {
         var provider = Provider.of<MemoryProvider>(context, listen: false);
```

Comment:
We allow pop, btw the sync process should be stop if users exited the sync page. We haven't had the background syncs yet 🫣

Reply:
The sync state is handled by provider and provider isn't disposed until the user closes the app. So the sync should keep running even if the user is on another screen. That is way changed the text as well (to only mention not to close the app)

Reply:
Lmk if I should revert it

Reply:
would be great if we could indicate it somewhere that the sync is running, and when the provider closes, we could gratefully stop the sync(e.g notify sd card to stop sync) @mdmohsin7 any idea?

Reply:
We can definitely have some sync in progress indicator maybe through a notification without cluttering the existing UI. What do you think? @beastoin 

Provider is only disposed when the app is restarted or the app is closed completely